sdk/eventgrid/Azure.Messaging.EventGrid/src/Customization/StorageDirectoryDeletedEventData.cs

```diff
@@ -0,0 +1,32 @@
+﻿// Copyright (c) Microsoft Corporation. All rights reserved.
+// Licensed under the MIT License.
+
+using Azure.Core;
+
+namespace Azure.Messaging.EventGrid.SystemEvents
+{
+    /// <summary> Schema of the Data property of an EventGridEvent for a Microsoft.Storage.DirectoryDeleted event. </summary>
+    public partial class StorageDirectoryDeletedEventData
+    {
+        [CodeGenMember("Recursive")]
+        internal string RecursiveString { get; }
+
+        /// <summary>
+        /// Is this event for a recursive delete operation.
+        /// </summary>
+        public bool? Recursive
```

Comment:
Should we consider `IsRecursive` here to follow typical .NET conventions?

Reply:
Unfortunately, this is already GA'ed - we are not changing the API here.